### ISSUE TITLE
human or not? a proposal for a reCAPTCHA step on all psiturk.org ads

### ISSUE THREAD

**Issue by gureckis** [NYUCCL/psiTurk#189]
I'm going to throw this idea out there...  add reCAPTCHA to all psiTurk ads after the worker accepts.  This way you can be more confident that the worker is not a bot.  While the instructions in most people's psiTurk tasks are sort of complex, it's still possible the certain psiTurk apps get exploited when they are open-sourced.  At the base level I can (maybe) easily add a reCAPTCHA step to every single ad listing to verify the worker entering the task is a real human.  

Thoughts?  Is that irritating?  My goal is 0 work for everyone but it could be something people don't want for some reason.

**Comment by twiecki** [comment]
That makes sense to me, and I don't think it would be irritating since captchas are pretty common. Is there evidence that this ever happened though? In any case, could always make it configurable.

**Comment by jodeleeuw** [comment]
I like the idea of having this on as a default option, but with the ability to turn it off for whatever reason.

**Comment by jbmartin** [comment]
+1

**Comment by JSlote** [comment]
I agree with Josh; an example of a time where one might wish to override it is
the experiment I currently have listed on the experiment exchange: a spoken word
recognition task that uses an audio-only reCaptcha to verify the user's hearing
ability.

On Fri, Jul 10, 2015 at 9:38 AM, Jay B. Martin < notifications@github.com [notifications@github.com] > wrote:
+1

—
Reply to this email directly or view it on GitHub
[https://github.com/NYUCCL/psiTurk/issues/189#issuecomment-120438506] .[https://github.com/notifications/beacon/AB8uYdqZFZcup3p-7oFP2r2zICntQA6Iks5ob96IgaJpZM4FVy2W.gif]
